pkgs/applications/version-management/pycalver/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, python3, git, mercurial}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pycalver";
+  version = "202010.1043";
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    sha256 = "6a194b9b14c8c81a7dbefb0659de4a156c49bf513f3bdb1aba49b371fff778d9";
+  };
+
+  checkInputs = [ python3.pkgs.pytestCheckHook git mercurial];
+  pytestFlagsArray = [ "--ignore=test/test_config.py" "--ignore=test/test_rewrite.py"];
+
+  propagatedBuildInputs = with python3.pkgs; [ pathlib2 click toml six setuptools ];
+
+  meta = with lib; {
+    description = "Bump version numbers in project files";
+    homepage = "https://pypi.org/project/pycalver/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ kfollesdal ];
+  };
+}
```

```suggestion
}

```

Missing final new line.